components/multiselect/overview.md

```diff
@@ -1,202 +1,142 @@
+
+## Creating MultiSelect
+
+1. Use the `TelerikMultiSelect` tag to add the component to a view, for example, `~/Pages/Index.razor`.
+
+1. Populate the `Data` property with the collection of items that you want to appear in the dropdown.
+
+1. Set the [two-way value binding]({%slug get-started-value-vs-data-binding %}#value-binding) by using `@bind-Value`.
+
+1. (Optional) Enable features like placeholder text, clear button, and AutoClose.
+
+>caption MultiSelect two-way value binding, main features, and simple [data binding](data-bind)
+
+````CSHTML
+@* Main features and simple data binding for the suggestions and the Value *@
+
+<TelerikMultiSelect Data="@Countries"
+                    @bind-Value="@Values"
+                    Placeholder="Enter Balkan country, e.g., Bulgaria"
+                    Width="350px" ClearButton="true" AutoClose="false">
+</TelerikMultiSelect>
+
+@if (Values.Count > 0)
+{
+    <ul>
+        @foreach (var item in Values)
+        {
+            <li>@item</li>
+        }
+    </ul>
+}
+
+@code {
+    List<string> Countries { get; set; } = new List<string>();
+    List<string> Values { get; set; } = new List<string>();
+
+    protected override void OnInitialized()
+    {
+        Countries.Add("Albania");
+        Countries.Add("Bosnia & Herzegovina");
+        Countries.Add("Bulgaria");
+        Countries.Add("Croatia");
+        Countries.Add("Kosovo");
+        Countries.Add("North Macedonia");
+        Countries.Add("Montenegro");
+        Countries.Add("Serbia");
+        Countries.Add("Slovenia");
+
+        base.OnInitialized();
+    }
+}
+````
+
+## Data Binding
+
+The Blazor MultiSelect requires a data source so that it can populate the dropdown with data. To provide a data source, use the `Data` property. [Read more about the Blazor MultiSelect data binding...]({% slug multiselect-databind %})
+
+## Filter
+
+The Blazor MultiSelect has a built-in filter that narrows down the shown suggestions as the end-user types. To configure this feature, use the `Filterable` parameter. Additionally, you can choose between different filter operators and configure after how many symbols the list with suggestions will appear. [Read more about the Blazor MultiSelect filter...]({% slug multiselect-filter %})
+
+## Grouping
+
+The Blazor MultiSelect enables you to group the listed suggestions into categories so you can help the end-user to browse faster through longer lists. [Read more about the Blazor MultiSelect grouping...]({% slug components/multiselect/grouping %})
+
+## Pre-Selecting Items

```

Comment:
Item pre-selection is not a feature of our components, but a Blazor feature, we should remove that from here IMO.